test/negative/SPV_INTEL_token_type/token_type_intel_negative.ll

```diff
@@ -0,0 +1,39 @@
+; Check whether the translator reports an error for a module with token type
```

Comment:
Does it make sense to have SPV_INTEL_token_type directory with only one file? Should we have just `test/negative/token_type.ll`?